Dockerfile

```diff
@@ -9,12 +10,22 @@ WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
 
 # install and cache app dependencies
-COPY ./client/package.json /app/package.json
-COPY ./client/package-lock.json /app/package-lock.json
-RUN npm install
-
```

Comment:
i'd still install the deps before copying the src so it doesn't do it each time the src changes but not the deps.

Reply:
Good point! Didn't think about that